packages/itwin/grouping-mapping-widget/api/grouping-mapping-widget.api.md

```diff
@@ -0,0 +1,522 @@
+import { StagePanelLocation } from '@itwin/appui-react';
+import { StagePanelSection } from '@itwin/appui-react';
+import type { UiItemsProvider } from '@itwin/appui-react';
+import { UseMutateAsyncFunction } from '@tanstack/react-query';
+import type { Widget } from '@itwin/appui-react';
+
+// @public
+export const CalculatedPropertyAction: ({ mappingId, group, calculatedProperty, onSaveSuccess, onClickCancel, }: CalculatedPropertyActionProps) => JSX.Element;
+
+// @public
+export interface CalculatedPropertyActionProps {
+    // (undocumented)
+    calculatedProperty?: CalculatedProperty;
+    // (undocumented)
+    group: Group;
+    // (undocumented)
+    mappingId: string;
+    // (undocumented)
+    onClickCancel?: () => void;
+    // (undocumented)
+    onSaveSuccess: () => void;
+}
+
+// @public
+export const CalculatedPropertyActionWithVisuals: ({ mappingId, group, calculatedProperty, onSaveSuccess, onClickCancel, }: CalculatedPropertyActionWithVisualsProps) => JSX.Element;
+
+// @public
+export interface CalculatedPropertyActionWithVisualsProps {
+    // (undocumented)
+    calculatedProperty?: CalculatedProperty;
+    // (undocumented)
+    group: Group;
+    // (undocumented)
+    mappingId: string;
+    // (undocumented)
+    onClickCancel?: () => void;
+    // (undocumented)
+    onSaveSuccess: () => void;
+}
+
+// @public (undocumented)
+export type ClientPrefix = "" | "dev" | "qa" | undefined;
+
+// @public
+export interface ContextCustomUI extends IGroupingMappingCustomUI {
+    onClick?: (group: Group, mapping: Mapping, iModelId: string) => void;
+    type: GroupingMappingCustomUIType.Context;
+    uiComponent?: React.ComponentType<ContextCustomUIProps>;
+}
+
+// @public
+export interface ContextCustomUIProps {
+    groupId: string;
+    iModelId: string;
+    mappingId: string;
+}
+
+// @internal (undocumented)
+export const createDefaultMappingClient: (prefix?: ClientPrefix) => IMappingsClient;
+
+// @internal (undocumented)
+export const createMappingClient: (clientProp: IMappingsClient | ClientPrefix) => IMappingsClient;
+
+// @public
+export const CustomCalculationAction: ({ mappingId, groupId, customCalculation, onSaveSuccess, onClickCancel, }: CustomCalculationActionProps) => JSX.Element;
+
+// @public
+export interface CustomCalculationActionProps {
+    // (undocumented)
+    customCalculation?: CustomCalculation;
+    // (undocumented)
+    groupId: string;
+    // (undocumented)
+    mappingId: string;
+    // (undocumented)
+    onClickCancel?: () => void;
+    // (undocumented)
+    onSaveSuccess: () => void;
+}
+
+// @public (undocumented)
+export type DataType = "Number" | "String" | "Boolean";
+
+// @public (undocumented)
+export type GetAccessTokenFn = () => Promise<AccessToken>;
+
+// @public
+export const GroupAction: (props: GroupActionProps) => JSX.Element;
+
+// @public
+export interface GroupActionProps {
+    // (undocumented)
+    displayStrings?: Partial<typeof defaultDisplayStrings_2>;
```

Comment:
I don't see `defaultDisplayStrings_2` defined anywhere.

Reply:
I don't want to export it. I guess `api-extractor` does not parse utilities.